scripts/perfmon

```diff
@@ -354,8 +352,8 @@ def average(mylist):
 
 def get_percent_log_fs_usage(ignored):
     "Get the percent usage of the host filesystem for logs partition. Input list is ignored and should be empty"
-    fs_output = commands.getoutput('df /etc/passwd')
-    log_fs_output = commands.getoutput('df /var/log')
+    fs_output = subprocess.getoutput('df /etc/passwd')
```

Comment:
do you expect string or byte result?

Reply:
Both the original code in python2 and the modified code in python3 returns a string
https://www.programcreek.com/python/example/599/commands.getoutput
https://docs.python.org/3/library/subprocess.html